### ISSUE TITLE
Trigger a Google crawl for Lux

### ISSUE THREAD

**Issue by nikdragovic** [emory-libraries/dlp-lux#503]
Once HTTP auth is removed, trigger a crawl via Google Search Console or Emory-preferred workflow.

What team is responsible for this process, and can be consulted in the future?
Is it possible to submit a Blacklight sitemap, and is anyone aware of the status on community developments?

**Comment by eporter23** [comment]
Looks like we need to add a verification code for Google webmaster in our DNS configuration.  I have a generated a code which can be [referenced in Box](https://emory.box.com/s/aw3lf0uqckzn22g12a4xfkw3139ows1d).

**Comment by libdgg** [comment]
I'm checking on the logistics of this. -Doug

**Comment by libdgg** [comment]
I am verifying through the LTDS google account where we have all the other web properties which means the site verification code is different than the one in Box. I have a ticket in for the verification via DNS request INC03371987

**Comment by libdgg** [comment]
@eporter23 I have an update on this that I need to run by you and the team. I'll add it to the dlp-launch ticket for our change management meeting discussion. -DG

**Comment by libdgg** [comment]
I closed/canceled the ticket INC03371987 since I need to get with Emily as a next step to determine if we want to proceed with DNS verification of "library.emory.edu" + URL verification for "digital.library.emory.edu" or approach this in another way. I'll resubmit a new ticket once we determine if we are using DNS verification or not.

**Comment by libdgg** [comment]
FYI for further discussion about Google Search Console / Google Analytics work

I see our analytics setup for digital.library.emory.edu here
https://analytics.google.com/analytics/web/?authuser=1#/report-home/a164499118w39366856p39046093

**Comment by mark-dce** [comment]
@libdgg from what I can see, if you have the top-level domain confirmed, you can ask google to crawl any subdomain without needing to re-validate the domain.  I just tried with our console and here's the steps I'm seeing:

**OPEN THE GOOLE SEARCH CONSOLE**
Here's the main console for our top-level domain - curationexperts.com
<img width="1120" alt="image" src="https://user-images.githubusercontent.com/3064318/80410785-bfa2b100-8890-11ea-85c0-921d7e5c4514.png">

**ENTER THE URL FOR THE SITE WITHIN YOUR DOMAIN YOU WANT TO INDEX**
tenejo is our demo Hyrax repository, it's reached as a subdomain of our primary domain (like digital is a subdomain of library.emory.edu) NOTE: you will need to wait to do this until HTTP Authentication is turned off
<img width="1120" alt="image" src="https://user-images.githubusercontent.com/3064318/80410965-02fd1f80-8891-11ea-8acb-85746f3fd10a.png">

**REQUEST REINDEXING FROM THE URL INSPECTION PAGE**
<img width="1120" alt="image" src="https://user-images.githubusercontent.com/3064318/80411169-57080400-8891-11ea-95d6-3b96f980b733.png">

<img width="1072" alt="image" src="https://user-images.githubusercontent.com/3064318/80411389-bfef7c00-8891-11ea-9521-cd184f67f12e.png">

**CHECK BACK ON PROGRESS PERIODICALLY**
Asking google to index the site without a sitemap will take some time, so you'll want to check in periodically to make sure no issues are encountered.
<img width="1072" alt="image" src="https://user-images.githubusercontent.com/3064318/80411389-bfef7c00-8891-11ea-9521-cd184f67f12e.png">

<img width="1120" alt="image" src="https://user-images.githubusercontent.com/3064318/80411464-dd244a80-8891-11ea-95a1-61eea944e0e1.png">

**Comment by mark-dce** [comment]
You can ensure a more thorough indexing by periodically submitting a full sitemap - there's some breadcrumbs here to automate sitemap generation https://github.com/projectblacklight/blacklight/wiki/Search-engine-harvesting

The blacklight list on google groups or the Code4Lib slack organization might be able to point you to more sample code for automating the submission of the sitemap to google.

**Comment by mark-dce** [comment]
@libdgg To see more detail about google's indexing, you'll want to add the site as persistent web 'property':

**SEE WHICH SITES ARE CONFIGURED**
<img width="1120" alt="image" src="https://user-images.githubusercontent.com/3064318/80412742-e9110c00-8893-11ea-98b4-c274fdf36b23.png">

**ADD A NEW PROPERTY (SITE) WITHIN YOUR DOMAIN**
<img width="1120" alt="image" src="https://user-images.githubusercontent.com/3064318/80412844-165dba00-8894-11ea-9cc6-16b1b6d048da.png">

**ADD SUBDOMAINS USING THE URL PREFIX**
<img width="1120" alt="image" src="https://user-images.githubusercontent.com/3064318/80412948-3d1bf080-8894-11ea-9e4f-d1502c7ac169.png">

For example, if we wanted to start indexing our dev site...
<img width="1120" alt="image" src="https://user-images.githubusercontent.com/3064318/80413007-591f9200-8894-11ea-8904-2628a4e917a6.png">

Because you (Emory's Google Search account) already owns the parent domain, you should be good to go
<img width="1120" alt="image" src="https://user-images.githubusercontent.com/3064318/80413095-88ce9a00-8894-11ea-86c3-dadc3dfcf84a.png">

**CHECK OUT INDEXING COVERAGE FOR YOUR SITE**
Notice that we recently cleaned out a lot of sample items that folks had created over time during conferences & workshops - hence the high excluded count
<img width="1120" alt="image" src="https://user-images.githubusercontent.com/3064318/80413312-e8c54080-8894-11ea-802f-aed8d8a4870f.png">

**Comment by libdgg** [comment]
Thanks Mark. I have submitted ticket INC03376253 to have the Emory DNS team verify the primary domain "library.emory.edu". Once that is done I plan to add the "digital.library.emory.edu" property per the info above from Mark.

**Comment by libdgg** [comment]
DNS verification complete for "digital.library.emory.edu" and "library.emory.edu". The Google Search Console shows both properties so it looks like everything worked.

**Comment by tmiles2** [comment]
@libdgg Can we close this ticket?
